hack/tools/release/notes.go

```diff
@@ -214,11 +229,65 @@ func run() int {
 
 	var commitRange string
 	var cmd *exec.Cmd
-
+	var lastDay string
+	var err error
+
+	if len(os.Args) < 2 {
+		fmt.Println("Usage: release-notes-generator [notes|weekly|help]")
+		fmt.Println("  notes: Generate release notes")
+		fmt.Println("  weekly: Generate weekly update notes")
+		fmt.Println("  help: Print this help message")
+		return 0
+	}
+
+	subcommand := os.Args[1]
+
+	weeklyCmd := flag.NewFlagSet(weeklySubCommand, flag.ExitOnError)
+	weeklyCmd.StringVar(since, "since", "", "Include commits starting from and including this date. Accepts format: YYYY-MM-DD")
+	weeklyCmd.StringVar(until, "until", "", "Include commits up to and including this date. Accepts format: YYYY-MM-DD")
+	weeklyCmd.StringVar(milestone, "milestone", "v1.4", "Milestone. Accepts format: v1.4")
+	weeklyCmd.StringVar(projectURL, "project-url", "https://github.com/kubernetes-sigs/cluster-api", "The project URL.")
```

Comment:
Sorry for my lack of clarity here - let's have this match the `repo` variable. We can assume `github.com` at this point, the same way we do in the nodesCmd.